attributes/packer_pipeline.rb

```diff
@@ -0,0 +1,6 @@
+default['osl-jenkins']['packer_pipeline']['openstack_taster_version'] = '0.0.2'
+default['packer']['version'] = '1.0.2'
```

Comment:
How do we determine what goes in the recipes and what goes in the attribute files ?